doc/02-Installation.md

```diff
@@ -0,0 +1,125 @@
+
+The recommended way to install Icinga Notifications is to use prebuilt packages
+for all supported platforms from our official release repository.
+Please follow the steps listed for your target operating system,
+which guide you through setting up the repository and installing Icinga Notifications.
+
+To upgrade an existing Icinga Notifications installation to a newer version,
+see the [Upgrading](04-Upgrading.md) documentation for the necessary steps.
+
+Before installing Icinga Notifications, make sure you have installed [Icinga 2](https://icinga.com/docs/icinga-2)
+with the [`api` feature](https://icinga.com/docs/icinga-2/latest/doc/12-icinga2-api/) configured and enabled.
+The Icinga 2 installation documentation covers all the necessary steps.
+
+<!-- {% else %} -->
+
+## Setting up the Database
+
+A MySQL (≥5.5), MariaDB (≥10.1), or PostgreSQL (≥9.6) database is required to run Icinga Notifications.
+Please follow the steps listed for your target database,
+which guide you through setting up the database and user and importing the schema.
+
+### Setting up a MySQL or MariaDB Database
+
+If you use a version of MySQL < 5.7 or MariaDB < 10.2, the following server options must be set:
+
+```
+innodb_file_format=barracuda
+innodb_file_per_table=1
+innodb_large_prefix=1
+```
+
+Set up a MySQL database for Icinga Notifications:
+
+```
+# mysql -u root -p
+
+CREATE DATABASE icinga-notifications;
+CREATE USER 'icinga-notifications'@'localhost' IDENTIFIED BY 'CHANGEME';
+GRANT ALL ON icinga-notifications.* TO 'icinga-notifications'@'localhost';
+```
+
+After creating the database, import the Icinga Notifications schema using the following command:
+
+```
+mysql -u root -p icinga-notifications </usr/share/icinga-notifications/schema/mysql/schema.sql
+```
+
+### Setting up a PostgreSQL Database
+
+Set up a PostgreSQL database for Icinga Notifications:
+
+```
+# su -l postgres
+
+createuser -P icinga-notifications
+createdb -E UTF8 --locale en_US.UTF-8 -T template0 -O icinga-notifications icinga-notifications
+psql icinga-notifications <<<'CREATE EXTENSION IF NOT EXISTS citext;'
+```
+
+The `CREATE EXTENSION` command requires the `postgresql-contrib` package.
+
+Edit `pg_hba.conf`, insert the following before everything else:
+
+```
+local all icinga-notifications           md5
+host  all icinga-notifications 0.0.0.0/0 md5
+host  all icinga-notifications      ::/0 md5
+```
+
+To apply these changes, run `systemctl reload postgresql`.
+
+After creating the database, import the Icinga Notifications schema using the following command:
+
+```
+psql -U icinga-notifications icinga-notifications < /usr/share/icinga-notifications/schema/pgsql/schema.sql
+```
+
+## Configuring Icinga Notifications
+
+Icinga Notifications installs its configuration file to `/etc/icinga-notifications/config.yml`,
+pre-populating most of the settings for a local setup. Before running Icinga Notifications,
+adjust the database credentials and, if necessary, the Icinga Web 2 URL.
```

Comment:
I'd omit "if necessary". I doubt there are scenarios where you wouldn't want to set it?